README.md

```diff
@@ -13,12 +13,12 @@
     > **spring-cloud-azure-starter-integration-servicebus**: The dependency that the sample depends on.
 
 ## Branch Policy
-We use `main` branch as the develop branch while setting the default branch as the latest released branch.
+Before Spring Cloud Azure 4.5.0, we use `main` branch as the develop branch while setting the default branch as the latest released branch.
+Now we use main branch as the latest released branch and set as default branch.
 
 | Branch Name               | Default Branch | Branch type    |
 |---------------------------|----------------|----------------|
-| main                      | false          | develop branch |
-| spring-cloud-azure_v4.5.0-beta.1 | true           | release branch |
+| main                      | true           | release branch |
```

Comment:
Should we add all branches for the major versions here, such as branch `feature/spring-boot-3`? it's better to add a version mapping description for Spring support.

Reply:
since there is no much differences between spring cloud azure 4 and 6, so the principle here is:

**Using one doc and sample for both versions to simplify work**